paramiko/kex_ecdh_nist.py

```diff
@@ -58,9 +64,9 @@ def _generate_key_pair(self):
 
     def _parse_kexecdh_init(self, m):
         Q_C_bytes = m.get_string()
-        self.Q_C = ec.EllipticCurvePublicNumbers.from_encoded_point(
+        self.Q_C = ec.EllipticCurvePublicKey.from_encoded_point(
```

Comment:
Based on the code in `_generate_key_pair` I'm pretty sure `self.Q_C` should be a key, not a numbers